lib/full-site-editing/edit-site-page.php

```diff
@@ -89,19 +89,17 @@ function gutenberg_edit_site_init( $hook ) {
 	 */
 	$current_screen->is_block_editor( true );
 
-	$settings = array_merge(
-		gutenberg_get_default_block_editor_settings(),
```

Comment:
`gutenberg_get_default_block_editor_settings` will be merged in the`gutenberg_get_block_editor_settings` function, which is why we remove it from here.

https://github.com/WordPress/gutenberg/blob/c96feebc4dafa087b5d247ad9c3a6ace5cb96d3c/lib/compat/wordpress-5.8/block-editor.php#L264-L272